platform/kube/inject/testdata/auth.non-default-service-account.yaml.injected

```diff
@@ -60,6 +63,9 @@ spec:
       - configMap:
           name: istio
         name: istio-config
+      - emptyDir:
+          medium: Memory
```

Comment:
This is to mount the envoy config directory as tmpfs (an in-memory file system).